public/static/locales/en/form-builder.json

```diff
@@ -18,6 +18,19 @@
   "remove": "Remove",
   "create": "Create",
   "json": "JSON",
-  "preview": "preview",
-  "placeHolderFormTitle": "Form Title"
+  "preview": "Preview",
+  "placeHolderFormTitle": "Form Title",
+  "share": "Share",
+  "save": "Save",
+  "saveH1": "Save your progress",
+  "saveP1": "We don’t save your form on our website, but don’t worry, you can save the file to your computer.",
+  "saveP2": "Save the from file to save your progress, or if you’re done, save it and open it again later to pick-up where you left off.",
```

Comment:
> Save the from file to save your progress

Save the form file ya?

Reply:
Yes, good point. I rewrote this sentence to be a bit simpler.